dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/ds3panel/Ds3PanelPresenter.java

```diff
@@ -683,6 +678,45 @@ public Label getPaneItemsLabel() {
     public Label getCreateNewSessionLabel() {
         return createNewSessionLabel;
     }
+
+    private void startGetJob(final List<Ds3TreeTableValueCustom> listFiles,
+            final Path localPath) {
+        listFiles.stream()
+                .map(Ds3TreeTableValueCustom::getBucketName)
+                .distinct()
+                .forEach(bucket -> {
+                    final ImmutableList<kotlin.Pair<String, String>> fileAndParent = listFiles.stream()
+                            .filter(ds3TreeTableValueCustom -> Objects.equals(ds3TreeTableValueCustom.getBucketName(), bucket))
+                            .map(ds3 -> new kotlin.Pair<>(
```

Comment:
It isn't real obvious to me what a ds3 is.  Can we have a more descriptive name?